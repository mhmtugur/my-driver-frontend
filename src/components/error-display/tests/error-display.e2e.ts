import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';

describe('error-display', () => {
  let page: E2EPage;
  let element: E2EElement;
  beforeEach(async () => {
    page = await newE2EPage({ html: '<error-display></error-display>' });
    element = await page.find('error-display');
  });

  it('should error', async () => {
    element.setProperty('error', { errorConstant: 'Newtork Error', errorMessage: 'Error occured' });
    await page.waitForChanges();
    expect(element.innerHTML).toMatchSnapshot();
  });
});