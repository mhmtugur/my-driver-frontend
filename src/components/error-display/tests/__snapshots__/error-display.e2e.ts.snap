// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`error-display should error 1`] = `"<div class=\\"error-display\\"><span>Error: Newtork Error  Error occured</span></div>"`;
